src/components/program-settings.js

```diff
@@ -69,20 +70,22 @@ class ProgramSettings extends React.Component {
         specificSetting: {
             openDialog: false,
             name: '',
-            settingDownload: '',
-            settingDBTrimming: '',
-            teiDownload: '',
-            teiDBTrimming: '',
-            enrollmentDownload: '',
-            enrollmentDBTrimming: '',
-            enrollmentDateDownload: '',
-            enrollmentDateDBTrimming: '',
-            updateDownload: '',
-            updateDBTrimming: '',
-            eventsDownload: '',
-            eventsDBTrimming: '',
-            eventDateDownload: '',
-            eventDateDBTrimming: '',
+            settingDownload: SpecificSettingsDefault.settingDownload,
```

Comment:
Looks like you should be able to just spread SpecificSettingsDefault here instead of setting each prop individually.

```specificSetting: {
openDialog: false,
name: '',
...SpecificSettingsDefault
}```